packages/universe-components/src/table/DonationsTableSortable.tsx

```
@@ -119,11 +181,17 @@
           list.map(row => {
             return (
               <TableRow key={row.id} data-id={row.id}>
-                {row.getVisibleCells().map(cell => (
-                  <TableCell key={cell.id}>
-                    {flexRender(cell.column.columnDef.cell, cell.getContext())}
-                  </TableCell>
-                ))}
+                {row.getVisibleCells().map(cell => {
+                  const align = cell?.column?.id == 'amount' ? 'text-right' : ''
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)